README.md

```diff
@@ -1,6 +1,6 @@
 # Marigold Sidechain
 
-A sidechain to Tezos by Marigold focused on higher throughput.
+The Marigold Sidechain to Tezos is designed to increase throughput.
```

Comment:
```suggestion
A sidechain to Tezos by Marigold focused on higher throughput.
```
I apologize for my very confusing language, but the original sentence fragment actually made more sense than the proposed complete sentence.

Reply:
Gotcha, I will keep the original then. Thanks.